.github/ISSUE_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug fix
+about: Give us your problem, and your solution
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Self checklist**
```

Comment:
I'm not sure we need a checklist for issues, and if we add it, I would add it at the bottom

any opinions?

Reply:
I am not a big fan of this because I'd rather have an incomplete bug report than someone not reporting because they need to provide to much information or the template is too complex

Reply:
The goal is not to discard any issue that wouldn't have filled up every information or checked every cases, this is more to encourage people to do so.  Of course if something is not relevant the person filling the issue doesn't need to make something up to declare.

Reply:
And I think it is more visible at the beginning of the template, if it is too far down people might miss it (even though this is the easiest part of the issue to fill in)

Reply:
if the template is so long, that people miss any part of it, it is simply too long.

(adding more to the template is not for free, in the contrary, every additional line adds a cost to any ticket filed, for every line we must consider, is what it adds worth it compared to what it cost.)

I think this self check is a bit hostile towards people filing a bug. There are not here to do something (as opposed to a PR), but just to communicate something. The template should help them, but in a good way.

Reply:
I will not approve a template with a checklist, this is way to hostile

Reply:
Removed it in https://github.com/inmanta/inmanta/pull/2416/commits/35fb7151fd7c0a968dadf87696f42179d272e650